docs/source/contributing.rst

```diff
@@ -0,0 +1,187 @@
+Contributing to Kordac
+#######################################
+
```

Comment:
Possibly want to provide an introduction about contributing, here is an excerpt from CSFG (some parts may be useful):

> Welcome to the Computer Science Field Guide (CSFG) developer community! We have spent many years creating the CSFG, and we would love for you to get involved into making this guide as great as possible! The following is a set of guidelines for contributing to the CSFG hosted on GitHub. These are just guidelines, not rules, use your best judgment and feel free to propose changes to this document in a pull request.

Probably should also mention Kordac will be/is used for conversion for two large two projects so Kordac needs to remain robust.